libs/core/device-manager/controllers/device-manager.controller.ts

```diff
@@ -51,7 +52,10 @@ export class DeviceManagerController {
     })
 
     if (result.ok) {
-      const deviceConfiguration = { caseColour: result.data.caseColour }
+      const deviceConfiguration = {
+        caseColour: result.data.caseColour,
+        serialNumber: result.data.serialNumber,
+      }
```

Comment:
how those changes connect to calculation of memory usage?

Reply:
Outdated